src/Schema/Events/Search.ts

```diff
@@ -235,3 +235,77 @@ export interface SelectedSearchSuggestionQuickNavigationItem {
   destination_path: string
   label: "Artworks" | "Auction Results"
 }
+
+/**
+ * A User fills shipping address and has auto completion with results
```

Comment:
This will most likely appear on multiple pages, so we might want to just remove the specific reference to shipping here and rely on context module.